### DISCUSSION TITLE
hello.qmd render error: Bad file descriptor

### DISCUSSION THREAD

**Discussion by SebasMaya** [quarto-dev/quarto-cli#6847] [Q&A] [answered]
### Description

Im just starting with VSCODE and Python, using Windows, after creating hello.qmd from the online tutorial I get the following error:

Starting python3 kernel...Done

Executing 'hello.ipynb'
  Cell 1/1...Done
ERROR: 

[Errno 9] Bad file descriptor

If I click the Run Cell the preview shows the polar figure.

How to enable preview of the whole document? Also render HTML fails with same error

**Comment by cscheid** [comment, marked as answer]
Are you running quarto on a network drive like OneDrive? This is a filesystem error, something in your system is unexpectedly closing a file under us. We don't recommend running quarto in network drives because this kind of error comes up often.

**Reply by SebasMaya** [reply]
Yes, it's in my documents folder ant that is synced with one drive. Will try to run in another folder. tks

**Reply by SebasMaya** [reply]
Moving to another folder solved the issue. Tks

**Comment by cscheid** [comment]
@cderv, I wonder if we should be checking this: https://stackoverflow.com/a/43625628

> `%OneDrive%` if you have only one OneDrive client on your PC or for the default one. If not, `%OneDriveConsumer%` for the personal OneDrive and `%OneDriveCommercial%` for OneDrive for Business.

Quarto could (probably should!) detect this, check if it's being called from inside that directory, and then emit a warning.

**Reply by cderv** [reply]
I did not know about those variables. Yes it seems a good idea to detect thoses and warn. 

I opened
* https://github.com/quarto-dev/quarto-cli/issues/6895
